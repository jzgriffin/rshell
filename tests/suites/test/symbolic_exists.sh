mkdir test_exists_dir
touch test_exists_file
[ -e test_exists_dir ] && echo a
[ -e test_exists_file ] && echo b
[ -e test_not_exists_dir ] && echo c
[ -e test_not_exists_file ] && echo d
rm -rf test_exists_dir test_exists_file
