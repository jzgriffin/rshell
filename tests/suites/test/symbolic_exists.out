(True)
a
(True)
b
(False)
(False)
